### ISSUE TITLE
Upgrade Alpine to 3.18 to avoid known DNS timeout issues in k8s

### ISSUE THREAD

**Issue by hfreire** [0xERR0R/blocky#1242]
It seems k8s officially reports some conntrack race issues with UDP DNS [1]. If you see the related GitHub issue [2] they point the issue to a specific version of Alpine and musl [2]. By reading the release notes of musl 1.2.4 you get a good understanding of what was broken in prior musl versions [3].

I've noticed that block Docker image is built from Alpine 3.17 which has this defect. Would it be possible to make a new release with Docker image built from Alpine 3.18?

[1] https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/#motivation
[2] https://github.com/kubernetes/kubernetes/issues/56903#issuecomment-1542624036
[3] https://www.openwall.com/lists/musl/2023/05/02/1

**Comment by kwitsch** [comment]
Have you tried the current development version?  

It technically still uses the golang docker image but upgrades it to get the latest patches.
Technically it should be >=3.18 [Dockerfile](https://github.com/kwitsch/ziggoimg/blob/main/Dockerfile).

**Comment by hfreire** [comment]
You are right, down there we have the Alpine 3.18 and patched musl version.

I still continue to see my issue with the development version so the problem must reside on my setup/environment.

Will close the issue as it does not seem related with blocky at all. Thanks for the quick feedback.
